portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,30 +38,46 @@
 public class DataServlet extends HttpServlet {
   private final String NAME = "name";
   private final String GET_LOAD_COMMENT_QUANTITY = "loadcommentquantity";
+  private final String USER_EMAIL = "email"; 
 
   public static final String COMMENT = "comment";
   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    UserService userService = UserServiceFactory.getUserService();
+
+    //server side Get check: login required to load comments
```

Comment:
Super nit: consider adding a space after // or /* comments.

Reply:
> Ah forgot to mention, it would be good to always include Marti for review in the future for the record and his information as well. Thanks!

yeah I think he's on it. Thanks